app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
```

Comment:
I think just with `t` method should work here
```suggestion
      <%= t("errors.messages.not_saved",
```

Reply:
Good catch! Fixed in [`3a8b1ec` (#30)](https://github.com/suho/google-search-ruby/pull/30/commits/3a8b1ecffc4b6d94f7ad5903c24c98ec3158850b)